tests/test_readers.py

```diff
@@ -9,7 +9,7 @@
 
 def test_load_feed():
     feed = ptg.load_feed(fixture("amazon-2017-08-06"))
-    assert feed.stop_times.dtypes["stop_id"] == np.object
```

Comment:
I don't recall why this assertion was checking for a dtype of `np.object` instead of `np.unicode`. Perhaps is should be updated to `str`?

Reply:
Looks like any column that doesn't have an assigned converter in `config.py` stays as `object` as it's not getting parsed. Maybe it makes sense to add something in `Feed._convert_types` to default to `str` if it doesn't have an override in `config.json`,

Reply:
Ah ok. That can be addressed in the future since there doesn't seem to be a clear need for the change. Thanks for looking into it.